internal/services/compute/virtual_machine_scale_set_resource_test.go

```diff
@@ -4398,6 +4401,7 @@ SETTINGS
     type                       = "DockerExtension"
     type_handler_version       = "1.0"
     auto_upgrade_minor_version = true
+    automatic_upgrade_enabled  = true
```

Comment:
unfortunately this resource is feature-frozen and as such unfortunately we'll need to roll this back:

```suggestion
```